journal/modernized/01-preface.adoc

```diff
@@ -74,7 +74,7 @@ and openly triumphed over principalities and powers.
 Now the old covenant was to pass away, and the new covenant to be established,
 and the priesthood changed, by the great High Priest without sin,
 and higher than the heavens; made not after the law of a carnal commandment,
-but by the power of an endless life, a priest forever, after the order of Melchisedec.
+but by the power of an endless life, a priest forever, after the order of Melchizedek.
```

Comment:
It seems this is an older rendering of Melchizedek, or an English one...I changed it here but left it in the original. Also, I forgot I have to add the proofread changes while I"m doing this. Glad I remembered this early on in the book!! :)